tcf_website/static/reviews/new_review.js

```diff
@@ -114,10 +118,11 @@ jQuery(function($) {
                 }).appendTo("#semester");
             });
             return this;
-        });
-
-        // Enable semester selector, disable the following
-        $("#semester").prop("disabled", false);
+        })
+            .done(function() {
+                // Enable semester selector, disable the following
```

Comment:
```suggestion
                // Enable semester selector
```
Nothing following the semester selector